_posts/2024-10-11-intro.md

```diff
@@ -16,6 +16,7 @@ We meet on Thursdays at 7pm in the CS building [50.34](https://www.kit.edu/campu
     * [Intro Slides](/talks/2024-11-07-intro/intro-slides.pdf), [Web Slides](/talks/2024-11-07-web/web-slides.pdf)
  * 14.11.24: Reverse Engineering
  * 21.11.24: Binary Exploitation
+    * [Pwn Slides](/talks/2024-11-21-pwnintro/slides.pdf)
```

Comment:
Can we rename this to `pwn-slides.pdf`?
I know I'm also guilty of naming everything `slides.pdf` but in hindsight it's not a good idea to have 4 pdfs called slides.pdf ^^